### ISSUE TITLE
Embed live stream and link to channel

### ISSUE THREAD

**Issue by nikosd** [euruko2013/site#89]
Link for the iframe:

```
<iframe src="http://www.ustream.tv/embed/14982647" width="608" height="368" scrolling="no" frameborder="0" style="border: 0px none transparent;"></iframe>
```

and [this is the channel on ustream](http://www.ustream.tv/channel/euruko-2013)

**Comment by pagojo** [comment]
> Link for the iframe:
> 
> <iframe src="http://www.ustream.tv/embed/14982647" width="608" height="368" scrolling="no" frameborder="0" style="border: 0px none transparent;"></iframe>
> 
> and this is the channel on ustream
> 
> —
> Reply to this email directly or view it on GitHub.

Not ready yet chaps we're working on it

/pagojo

**Comment by fotos** [comment]
Closed by 60bd28f2fb7c9.
